### ISSUE TITLE
Explore inline word highlighting in info texts

### ISSUE THREAD

**Issue by hbons** [microsoft/vscode#188657]
From @isidorn:

<img width="258" alt="image" src="https://github.com/microsoft/vscode/assets/103326/e4542bfe-a23c-41b7-a83f-8632be47f66d">

Highlighting (variable) words improves readability by making the text easier to scan.
It would be useful to implement this as a more global feature.

cc @daviddossett

**Comment by daviddossett** [comment]
One idea here would be to align with `<code>` blocks like `alexr00/testLocalBranch`.

**Comment by hbons** [comment]
It seems that `markdown` backticks are already supported. We just need to style it nicely, document it, and use it in more places. :)

**Comment by hbons** [comment]
example: 

```
new vscode.MarkdownString(vscode.l10n.t('There are no commits between the base `{0}` branch and the comparing `{1}` branch', this.baseBranchName, this.compareBranchName));
```

**Comment by daviddossett** [comment]
Yeah, we have [default markdown styles](https://github.com/dnicolson/vscode/blob/732e36288a66bb90552541c8791122744bc21a0b/extensions/markdown-language-features/media/markdown.css#L209-L210) that don't quite look like that—but I'd like them to.
